spring-cloud-dataflow-server-kubernetes/pom.xml

```diff
@@ -83,6 +95,27 @@
 				  </images>
 				</configuration>
 			</plugin>
+			<plugin>
```

Comment:
doing a `touch spring-cloud-dataflow-server-kubernetes/.jdk8` seems to avoid having to add this. the `.jdk8` file is also present in the CF deployer where it looks like you didn't need to add this plugin